### PR TITLE
Fix dependencies in setup.py and keep version consistent

### PR DESCRIPTION
**Pull request by sonnyhu**
1. `setup.py` did not include `numpy` which is required, so it was not self-contained. CircleCI will run `setup.py` so there is no need to do `pip install`.
2.  `version` in `version.py` was inconsistent with tag. In our core, we install `chumpy==0.0.3`, and this comes from git tag.

### CODE REVIEW

File: version.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-version = '0.74'
+version = '0.0.4'
```
Comment (algrs) [inline version.py]: Why `0.0.4`? This is technically moving backwards in semver; we should probably go to `0.75.0`.

Comment (sonnyhu) [inline version.py]: We have `0.0.3` as latest tag.

Comment (sonnyhu) [inline version.py]: And we use `0.0.3` https://github.com/bodylabs/core/blob/master/requirements/requirements.in#L17

File: setup.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def setup_opendr(ext_modules):
             url = 'http://github.com/mattloper/opendr',
             ext_package='opendr',
             package_data={'opendr': ['test_dr/nasa*']},
-            install_requires=['Cython', 'chumpy >= 0.58', 'matplotlib'],
+            install_requires=['Cython', 'chumpy >= 0.58', 'matplotlib', 'numpy==1.10.1'],
```
Comment (algrs) [inline setup.py]: Really, this should be done by putting these in `requirements.txt` and reading it in here, like it's done in `skeleton-py`...

File: version.py (hunk 1 above)
Comment (algrs) [inline version.py]: Ok... So we screwed up before and tagged it with something other than its version number...

Let's change this to `0.75.0` and tag it with the same. That way we're not moving backwards (which would confuse anything that used `>=`) and we get the version and the tags back in sync.

**Comment by algrs** [comment]
👍

**Pull request merged**